docs/content/ingestion/index.md

```diff
@@ -100,8 +100,9 @@ This is for realtime ingestion. Make sure to include "io.druid.extensions:druid-
 | type | String | This should say `avro_stream`. | no |
 | avroBytesDecoder | JSON Object | Specifies how to decode bytes to Avro record. | yes |
 | parseSpec | JSON Object | Specifies the format of the data. | yes |
+| schemaMappings | JSON Object | Specifies a mapping from field names found within the parseSpec to data within a complex avro structure. | no |
```

Comment:
@zhaown flattening is supported by the json parser in 0.9.0 (actually 0.8.3 too but it wasn't documented there). See http://druid.io/docs/0.9.0-rc1/ingestion/flatten-json.html for docs.

@himanshug @will-lauer if possible it'd be nice to support the same flattening spec for avro and json, unfortunately the avro schemaMappings here and the json flattenSpec already in master are not compatible. Do you guys have thoughts about which one is 'better' and whether we could choose one to work with both parsers?